src/eslint-config-adeira/package.json

```diff
@@ -21,7 +21,7 @@
     "eslint-plugin-adeira": "^0.15.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-fb-flow": "^0.0.4",
-    "eslint-plugin-ft-flow": "^1.1.0",
+    "eslint-plugin-ft-flow": "^1.1.1-alpha-0",
```

Comment:
TODO: wait for a proper release

Reply:
Released to [v2.0.0](https://github.com/flow-typed/eslint-plugin-ft-flow/releases/tag/v2.0.0)